.github/workflows/python-app.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         # Default builds are on Ubuntu
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9, pypy-3.7]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Python 3.10 has been released. Consider adding it to the list. 3.7 -> 3.10 are all currently supported. 3.6 is nearly end of life, so there's no need to support it.